### PR TITLE
Change labels to sentence case for PDF component

### PR DESCRIPTION
**Pull request by imsoumik03**
issue #7792 
### Description

_I have changed the values of the followings of the PDF component ->_

- displayName
- description
- url.displayName
- showDownloadOption.displayName

### CODE REVIEW

File: frontend/src/Editor/WidgetManager/widgetConfig.js [hunk 1]
```diff
@@ -4885,16 +4885,16 @@ ReactDOM.render(<ConnectedComponent />, document.body);`,
   },
   {
     name: 'PDF',
-    displayName: 'PDF',
-    description: 'Embed PDF documents',
+    displayName: 'Pdf',
```
Comment (tejakummarikuntla) [inline frontend/src/Editor/WidgetManager/widgetConfig.js:4888]: Please revert this change.

File: frontend/src/Editor/WidgetManager/widgetConfig.js [hunk 2]
```diff
@@ -4885,16 +4885,16 @@ ReactDOM.render(<ConnectedComponent />, document.body);`,
...
+    description: 'Embed pdf documents',
```
Comment (tejakummarikuntla) [inline frontend/src/Editor/WidgetManager/widgetConfig.js:4889]: Please revert this change.

File: frontend/src/Editor/WidgetManager/widgetConfig.js [hunk 3]
```diff
@@ -4885,16 +4885,16 @@ ReactDOM.render(<ConnectedComponent />, document.body);`,
...
     component: 'PDF',
     properties: {
-      url: { type: 'code', displayName: 'File URL', validation: { schema: { type: 'string' } } },
+      url: { type: 'code', displayName: 'File url', validation: { schema: { type: 'string' } } },
```
Comment (tejakummarikuntla) [inline frontend/src/Editor/WidgetManager/widgetConfig.js:4892]: Please revert this change.

**Review by tejakummarikuntla — CHANGES_REQUESTED** [review]
Hey @imsoumik03, kindly look into the comments and update the changes.

**Comment by tejakummarikuntla** [comment]
Hey @imsoumik03, kindly address the suggested changes to progress the PR.

**Comment by karan-rathod-316** [comment]
Hi @imsoumik03, reassigning this to someone else as we are not getting any response from you.

**Pull request closed without merging**